### PR TITLE
Add placeholder block cat_filter_extra (category overview)

### PR DESCRIPTION
**Pull request by marengeti**
A placeholder for an additional section into the Category Overview side_menu section -- for an additional filtering mini form.

### CODE REVIEW

File: indico/modules/categories/templates/display/overview/base.html [hunk 1]
```diff
@@ -43,6 +43,7 @@ <h3>{% trans %}Display options{% endtrans %}</h3>
                         </select>
                     </div>
                 </div>
+                {% block cat_filter_extra %}{% endblock %}
```
Comment (mic4ael) [inline indico/modules/categories/templates/display/overview/base.html:46]: Maybe the name could be improved a bit? `category_extra_filters`?

Comment (marengeti) [reply indico/modules/categories/templates/display/overview/base.html]: Sure, if you prefer that, done. I was unable to project a clear block naming convention from the block names I see?

Comment (mic4ael) [reply indico/modules/categories/templates/display/overview/base.html]: No need for a separate commit though ;) You can squash them into one :)

Comment (marengeti) [reply indico/modules/categories/templates/display/overview/base.html]: Can be squashed on merge,.

**Comment by marengeti** [comment]
Is there still some problem with this?

**Comment by ThiefMaster** [comment]
Nope, everything fine with it :)

**Comment by marengeti** [comment]
Thanks!

**Pull request merged**